core/src/main/scala/kafka/server/BrokerToControllerChannelManager.scala

```diff
@@ -211,7 +235,7 @@ class BrokerToControllerChannelManagerImpl(
         50,
         Selectable.USE_DEFAULT_BUFFER_SIZE,
         Selectable.USE_DEFAULT_BUFFER_SIZE,
-        config.requestTimeoutMs,
+        networkClientRetryTimeoutMs,
```

Comment:
This change is cosmetic only -- the value is never used by this code as currently implemented.